packages/components/app/styles/components/pagination.scss

```diff
@@ -10,19 +10,43 @@
 
 .hds-pagination {
   display: grid;
-  grid-template-areas: "left center right";
+  grid-template-areas: "info nav selector";
   grid-template-rows: auto;
   grid-template-columns: 1fr auto 1fr;
   align-items: center;
   margin: 0 auto;
 }
 
+// Responsive layout
+// max-width is based on where component layout potentially breaks down
+@media screen and (max-width: 1000px) {
+  .hds-pagination {
+    display: flex;
+    flex-direction: column;
+    height: auto;
+
+    .hds-pagination-info,
+    .hds-pagination-nav,
+    .hds-pagination-size-selector {
+      margin-right: auto;
+      margin-left: auto;
+    }
+  }
+
+  .hds-pagination-nav { order: -1; }
+
+  .hds-pagination-info,
+  .hds-pagination-size-selector {
+    margin-top: 8px;
```

Comment:
This value can be adjusted if desired and can be exposed as a component-level token (CSS custom property) before release.